kafka/Dockerfile

```diff
@@ -1,6 +1,4 @@
-# Share the same base image to reduce layers used in testing
-FROM openzipkin/jre-full:1.8.0_191
-MAINTAINER OpenZipkin "http://zipkin.io/"
+FROM debian:stretch-slim
```

Comment:
please add a comment about why this image

Reply:
I think I ended up trying debian here in many investigations into runit. But I was able to revert to alpine and be closer to the original code.